source/shared/cpp/ObjectModel/MarkDownParser.h

```diff
@@ -0,0 +1,94 @@
+#pragma once
+
+#include <iomanip>
+#include <list>
+#include "pch.h"
+#include "BaseCardElement.h"
+#include "Enums.h"
+
+namespace AdaptiveCards
+{
+class MarkDownParser
+{
+public:
+    MarkDownParser(const std::string &txt); 
+
+    enum DelimiterType
+    {
+        Init,
+        Alphanumeric,
+        Puntuation,
+        Escape,
+        WhiteSpace,
+        Underscore = 0x8,
+        Asterisk   = 0x10,
```

Comment:
Why are these set to these values?  It looks like maybe you want a bitmask here, but i didn't notice a place where this value was being used as a bitmask.  If you need bitmask values, lets have all the values be bitmask values, otherwise just let these be sequential.